tests/ext/segfault_backtrace_enabled.phpt

```diff
@@ -4,8 +4,11 @@ Dump backtrace when segmentation fault signal is raised and config enables it
 <?php
 if (!extension_loaded('posix')) die('skip: posix extension required');
 if (getenv('SKIP_ASAN') || getenv('USE_ZEND_ALLOC') === '0') die("skip: intentionally causes segfaults");
+if (getenv('DD_TRACE_CI_PECL') === '1') die("skip: pecl run-tests does not support XFAIL");
```

Comment:
This handles the case we see in CI when we run with the very old and terrible Pear test runner.

Reply:
We don't need to add a new env var here since we can check the existing `PHP_PEAR_RUNTESTS` ([example](https://github.com/DataDog/dd-trace-php/blob/f5666ab/tests/ext/sandbox/hook_function/01.phpt#L6)).

Reply:
Excellent, I went looking for something like that and couldn't find it in RunTest source code ... 

RunTests is very terrible by the wa, hasn't been updated in > ten years, and almost certainly shouldn't be part of anyones CI pipeline.